components/alibi-detect-server/adserver/od_model.py

```diff
@@ -82,13 +82,24 @@ def process_event(self, inputs: List, headers: Dict) -> Dict:
             and headers[HEADER_RETURN_FEATURE_SCORE] == "true"
         ):
             ret_feature_score = True
-        od_preds = self.model.predict(
-            X,
-            outlier_type=outlier_type,
-            # use 'feature' or 'instance' level
-            return_feature_score=ret_feature_score,
-            # scores used to determine outliers
-            return_instance_score=ret_instance_score,
-        )
+        op_preds = {}
+        name = self.model.meta['name']
```

Comment:
Would model meta name always be set? Otherwise may be worth adding .get("name", None) to avoid exception

Reply:
I think it should always be there: https://github.com/SeldonIO/alibi-detect/blob/ca9af48e3590121daff39a6c5d76261838867d3c/alibi_detect/utils/saving.py#L68

Reply:
Ah ok i seen, thanks for the reference